packages/database/src/migrations/20210415110921-UpdatePSSSWeeklyIndicatorsSupportSiteData-modifies-data.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const oldConfig = syndromeCode => ({
+  formula: `firstExistingValue(PSSS_${syndromeCode}_Cases, PSSS_${syndromeCode}_Daily_Cases)`,
+  aggregation: {
+    [`PSSS_${syndromeCode}_Cases`]: 'FINAL_EACH_WEEK',
+    [`PSSS_${syndromeCode}_Daily_Cases`]: ['SUM_EACH_WEEK', 'FINAL_EACH_WEEK'],
+  },
+  defaultValues: {
+    [`PSSS_${syndromeCode}_Cases`]: 'undefined',
+    [`PSSS_${syndromeCode}_Daily_Cases`]: 'undefined',
+  },
+});
+
+const siteParameters = syndromeCode => [
+  {
+    code: 'sumSiteWeeklyCases',
+    config: {
+      formula: `PSSS_${syndromeCode}_Cases`,
+      aggregation: [
+        'FINAL_EACH_WEEK',
+        {
+          type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+          config: {
+            dataSourceEntityType: 'facility',
+            aggregationEntityType: 'country',
```

Comment:
One might argue that we ought to use `requested` here, but given that we currently only care about aggregating to the country level I'd rather be explicit and we can change it later if needs be